nomad/structs/structs.go

```diff
@@ -7822,6 +7816,44 @@ func (t *Template) DiffID() string {
 	return t.DestPath
 }
 
+// ChangeScript holds the configuration for the script that is executed if
+// change mode is set to script
+type ChangeScript struct {
+	// Command is the full path to the script
+	Command string
+	// Args is a slice of arguments passed to the script
+	Args []string
+	// Timeout is the amount of seconds we wait for the script to finish
+	Timeout time.Duration
+	// FailOnError indicates whether a task should fail in case script execution
+	// fails or log script failure and don't interrupt the task
+	FailOnError bool
+}
+
+func (cs *ChangeScript) Copy() *ChangeScript {
+	if cs == nil {
+		return nil
+	}
+
+	ncs := new(ChangeScript)
+	*ncs = *cs
+
+	return ncs
+}
```

Comment:
Unfortunately you'll need to copy the `Args` field as well. Fortunately we have [`helper.CopySlice`](https://github.com/hashicorp/nomad/blob/main/helper/funcs.go#L289-L301). See https://play.golang.com/p/JAnuZGsPlBw

Reply:
uff, of course, thanks for catching this. fixed.